src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights.csproj

```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.2.0" />
```

Comment:
consider using 2.3. It is currently on MyGet https://www.myget.org/F/applicationinsights/api/v3/index.json and will be released soon to NuGet